lua/telescope/builtin/init.lua

```diff
@@ -1,16 +1,23 @@
 ---@tag telescope.builtin
 
 ---@brief [[
---- A collection of builtin pickers for telescope.
----
---- Meant for both example and for easy startup.
+--- A collection of builtin pickers for telescope, meant for both example and for easy startup.
```

Comment:
Maybe some rewording is needed here.

e.g. 

> Builtins are a collection of community maintained pickers to support common workflows. It can be used as reference when building prs, extension, or user pickers.

Reply:
> Builtins are a collection of community maintained pickers to support common workflows. It can be used as reference when writing PRs, Telescope extensions, your own custom pickers, or just as a discovery tool for all of the amazing pickers already shipped with Telescope!

Thoughts on this?

Reply:
Oh that's sound so much better. `Telescope Builtins is a collection of`..

Reply:
Done!